liquibase-standard/src/main/java/liquibase/snapshot/jvm/ColumnSnapshotGenerator.java

```diff
@@ -574,30 +575,32 @@ protected Object readDefaultValue(CachedRow columnMetadataResultSet, Column colu
 
         if ((database instanceof AbstractDb2Database)
                 && ((columnMetadataResultSet.get(COLUMN_DEF_COL) != null)
-                && StringUtil.equalsWordNull((String) columnMetadataResultSet.get(COLUMN_DEF_COL)))) {
+                && "null".equalsIgnoreCase((String) columnMetadataResultSet.get(COLUMN_DEF_COL)))) {
             columnMetadataResultSet.set(COLUMN_DEF_COL, null);
         }
 
         if (database instanceof SybaseASADatabase) {
             String defaultValue = (String) columnMetadataResultSet.get(COLUMN_DEF_COL);
+            if (defaultValue != null) {
 
-           // SQL Anywhere returns `CURRENT DATE` (without underscore), which no other RDBMS would understand
-           defaultValue = defaultValue.replaceAll("(?i)\\bCURRENT\\s+DATE\\b", "{fn CURDATE()}");
+               // SQL Anywhere returns `CURRENT DATE` (without underscore), which no other RDBMS would understand
+               defaultValue = defaultValue.replaceAll("(?i)\\bCURRENT\\s+DATE\\b", "{fn CURDATE()}");
 
-           // SQL Anywhere returns `CURRENT TIME` (without underscore), which no other RDBMS would understand
-           defaultValue = defaultValue.replaceAll("(?i)\\bCURRENT\\s+TIME\\b", "{fn CURTIME()}");
+               // SQL Anywhere returns `CURRENT TIME` (without underscore), which no other RDBMS would understand
+               defaultValue = defaultValue.replaceAll("(?i)\\bCURRENT\\s+TIME\\b", "{fn CURTIME()}");
 
-           // SQL Anywhere returns `CURRENT TIMESTAMP` (without underscore), which no other RDBMS would understand
-           defaultValue = defaultValue.replaceAll("(?i)\\bCURRENT\\s+TIMESTAMP\\b", "{fn NOW()}");
+               // SQL Anywhere returns `CURRENT TIMESTAMP` (without underscore), which no other RDBMS would understand
+               defaultValue = defaultValue.replaceAll("(?i)\\bCURRENT\\s+TIMESTAMP\\b", "{fn NOW()}");
 
-           // SQL Anywhere returns `CURRENT USER` (without underscore), which no other RDBMS would understand
-           defaultValue = defaultValue.replaceAll("(?i)\\bCURRENT\\s+USER\\b", "{fn USER()}");
+               // SQL Anywhere returns `CURRENT USER` (without underscore), which no other RDBMS would understand
+               defaultValue = defaultValue.replaceAll("(?i)\\bCURRENT\\s+USER\\b", "{fn USER()}");
+           }
 
            columnMetadataResultSet.set(COLUMN_DEF_COL, defaultValue);
 
            if (YES_VALUE.equals(columnMetadataResultSet.get(IS_GENERATED_COLUMN))) {
                Object virtColumnDef = columnMetadataResultSet.get(COLUMN_DEF_COL);
-               if ((virtColumnDef != null) && !StringUtil.equalsWordNull(virtColumnDef.toString())) {
+               if ((virtColumnDef != null) && !"null".equalsIgnoreCase(virtColumnDef.toString())) {
```

Comment:
I have a question about this change. Was it done because StringUtil is a deprecated class? If that's the reason behind this change I think we should revert it back as it was because we have an open PR to undeprecate `StringUtil` class.

Reply:
@MalloD12 Probably yes, because of deprecation.
BUT
I don't see any reasons to revert this change. 
Expression `"null".equalsIgnoreCase` is more short and absolutely clear. You don't need to look into documentation to quickly understand what id does.
Definitely better than `StringUtil.equalsWordNull`.

Reply:
Hi @asolntsev, 

I think it's maybe more a matter of preference.   In the past, we worked on an issue to make this type of replacement by the equivalent `StringUtil` methods (see #3718).

Reply:
Yes, but in our case we are not replacing `StringUtil` by `StringUtils`.
We are replacing `StringUtil` by a native Java method. Which is (at least) no longer or harder to read.

Reply:
I very much appreciate your intention to keep improving Liquibase code, but it was a team preference we decided to make in the past to update this kind of conditional as specified in #3718.

Thanks,
Daniel.

Reply:
Ok, I reverted all StringUtils-related changes and rebased the branch from the latest master.